pysingfel/experiment/fxs.py

```diff
@@ -9,10 +9,11 @@
 
 
 class FXSExperiment(Experiment):
-    def __init__(self, det, beam, particles, n_part_per_shot, ratios=None):
+    def __init__(self, det, beam, particles, n_part_per_shot, gamma, ratios=None):
```

Comment:
gamma should have a default value to be compatible with other scripts in /examples. I suggest gamma=0.